api/dscinitialization/v1/dscinitialization_conversion.go

```diff
@@ -40,9 +40,7 @@ func (c *DSCInitialization) ConvertTo(dstRaw conversion.Hub) error {
 	}
 	if c.Spec.DevFlags != nil {
 		dst.Spec.DevFlags = &dsciv2.DevFlags{
-			ManifestsUri: c.Spec.DevFlags.ManifestsUri,
-			LogMode:      c.Spec.DevFlags.LogMode,
```

Comment:
should this be cleaned up https://github.com/davidebianchi/opendatahub-operator/blob/b3345b377c971582262a44a859aaabfb0bfbaf9b/cmd/main.go#L169  and other places

Reply:
[This](https://github.com/davidebianchi/opendatahub-operator/blob/b3345b377c971582262a44a859aaabfb0bfbaf9b/cmd/main.go#L169) is taken from operator config, not from dev flags. Should I remove it? 
In the conversion are already deleted, I kept in DSCI v1 to avoid changes in that version. But as for the components, I could remove also there. What do you think?